spec/controllers/clients_controller_spec.rb

```diff
@@ -72,19 +72,43 @@
   end
 
   describe "GET #destroy" do
-    xit "deletes a note" do
+
+    it "destroys a note for a user" do
       login_user
-      updated_client = create(:client,
-                      user_id: subject.current_user.id)
-      note = create(:note,
-                    user_id: subject.current_user.id,
-                    client_id: client.id)
+      client = create(:client)
+      params = {
+        description: "Test description",
+        client_id: client.id
+      }
+
+      post :note_create, id: client.id, note: params
+      note = Note.last
+
       expect(Note.all.size).to eq(1)
       get :destroy, id: note.id
       expect(response).to redirect_to( client_path(client.id) )
       expect(Note.all.size).to eq(0)
       expect(subject.request.flash[:success].first).to eq("Note deleted.")
```

Comment:
Maybe you've already discussed this, but let's discuss as a group on Thursday.
Is better practice to have multiple expectations in a single assertion? Or should `each test should make only one assertion`? It may be more efficient this way.